python/alibiexplainer/pyproject.toml

```diff
@@ -17,6 +17,7 @@ tensorflow = ">=2.12.0,<2.14"  # the range that supports python 3.8 -- 3.11
 dill = "^0.3.6"
 nest-asyncio = "~1.4.0"
 llvmlite = ">0.38.1" # needed since poetry chooses lower version of llvmlite which is not supported by python 3.9 above
+tensorflow-io-gcs-filesystem = "0.34.0"
```

Comment:
Is this intentional?

Reply:
yes.
Otherwise, when running poetry lock it will get updated to 0.36.0 and the alibexplainer build will fail by not finding this dependency.